index.html

```diff
@@ -117,30 +117,41 @@ <h1 class="title">Publishing@W3C</h1>
                                 </p>
 							
-	                            <h2 class="cf">How to Participate</h2>
+                                <h2 class="cf">How to Participate</h2>
+
                                 <p>
-                                    We welcome global participation in Publishing@W3C activities from any organization interested in joining the conversation about the future of publishing. For those interested in maintaining EPUB 3 technical requirements and advancing further adoption, we encourage you to join the new <a href="https://www.w3.org/publishing/groups/epub3-cg/">EPUB 3 Community Group</a> which is free and open to all. Simply go there to sign up.
+                                    The focal point of general conversations for the future of publishing takes place in the <a href="https://www.w3.org/publishing/groups/publ-bg/">Publishing Business Group</a>; to join the Group, please go to the Group’s page to sign up. Participation in this group requires a modest fee. 
                                 </p>
+                                
                                 <p>
-                                    The focal point of general conversations for the future of publishing takes place in the <a href="https://www.w3.org/publishing/groups/publ-bg/">Publishing Business Group</a>; to join the Group, please go to the Group’s page to sign up. Participation in this group requires a modest fee. 
+                                    The present and future of EPUB lies by the <a href="https://www.w3.org/publishing/groups/epub-wg/">EPUB 3 Working Group</a>. This group works on ensuring the interoperability of EPUB; it is also the home for the technical evolution of EPUB 3 while maintaining a strong backward compatibility and taking into account the business needs of the publishing community. Participation in that group requires to be a regular W3C member; to join that group, please <a href="https://www.w3.org/2004/01/pp-impl/125377/join">sign up here</a>.  
+                                </p>
+
+                                <p>
+                                    The EPUB 3 Working Group is closely related to the <a href="http://www.w3.org/community/publishingcg/">Publishing Community Group</a>. While the EPUB 3 Working Group is the guardian of the EPUB standard, the Community Group is the home for discussions around new technical directions, ideas, and experimentation; some of these ideas may become part of a future EPUB or other publishing standards. It is the primary “incubation arm” or the of Publishing@W3C, the birthplace of new ideas for the future. We encourage you to join the new <a href="http://www.w3.org/community/publishingcg/">EPUB 3 Community Group</a> which is free and open to all. Simply go there to sign up.
```

Comment:
I don't seem to be able to edit directly here. one minor change:
s/ some of these ideas may become part of a future EPUB or other publishing standards. It is the primary “incubation arm” or the of Publishing@W3C/
some of these ideas may become part of a future EPUB or other standards. It is the primary “incubation arm” or the of Publishing@W3C

Reply:
Thanks, done it!